pandas/tests/util/test_testing.py

```diff
@@ -276,6 +276,19 @@ def test_numpy_array_equal_message(self):
             assert_almost_equal(np.array([[1, 2], [3, 4]]),
                                 np.array([[1, 3], [3, 4]]))
 
+        expected = """numpy array are different
```

Comment:
can you make a new test & indicate the gh issue with a comment

Reply:
👍 6b087f2